src/main/kotlin/com/exactpro/th2/common/schema/message/MessageRouterUtils.kt

```diff
@@ -71,4 +76,35 @@ fun appendAttributes(
     return mutableSetOf(*attributes).apply {
         addAll(requiredAttributes())
     }
+}
+
+fun MessageGroupBatch.toShortDebugString(): String {
+    val sessionAliases = mutableListOf<String>()
+    val directions = mutableListOf<Direction>()
```

Comment:
Why lists? We will get a lot of repeating values in the result if the batch contains all messages from the same session and direction

Reply:
Yep, you're right, my bad

Reply:
Maybe it would be better to print message id's instead?

Reply:
Do you mean all message IDS? If that so, I think it will be better but only in case we will form the message ID ourselves and won't use `toJson` method.
Something like: <session>:<direction>:<sequence>[.<subsequences>+]